### PR TITLE
Feature [Report Sharing, Layout Shift, Reusable Components]

### PR DESCRIPTION
**Pull request by sarkartanmay393**

### CODE REVIEW

**Comment by sarkartanmay393** [comment]
@Nil2000 Please review this and let me know.

File: src/app/layout.tsx [hunk 1]
```diff
@@ -14,10 +18,15 @@ export const metadata: Metadata = {
 export default function RootLayout({
   children,
 }: Readonly<{ children: React.ReactNode }>) {
+
   return (
     <html lang="en" className={`${GeistSans.variable}`}>
-      <body className="relative scrollbar-thin scroll-smooth">
+      <body className="relative scrollbar-thin scroll-smooth flex min-h-screen flex-col items-center justify-center">
```
Comment (Nil2000) [inline src/app/layout.tsx:24]: I checked this `smooth-scroll` does not work. Is it?
Also `scrollbar-thin` is not working.

Comment (Nil2000) [reply src/app/layout.tsx:24]: Probably need to add this `class` inside `<html>` classname

Comment (sarkartanmay393) [reply src/app/layout.tsx:24]: Yes you are right. Let me check this on next commit.

**Pull request merged**